### PR TITLE
Added onSizeChange prop so that parent can be notified of size changes

### PR DESCRIPTION
**Pull request by ndbroadbent**
This depends on 293208c, which fixes a race condition with the original `onSizeChange` callback. I included that commit in #88

### CODE REVIEW

File: RNAdMobBanner.js [hunk 1]
```diff
@@ -29,7 +32,7 @@ export default class AdMobBanner extends React.Component {
       <View style={this.props.style}>
         <RNBanner
           style={this.state.style}
-          onSizeChange={this.onSizeChange.bind(this)}
```
Comment (ndbroadbent) [inline RNAdMobBanner.js]: These are already bound in the constructor.

**Comment by ndbroadbent** [comment]
The reason for this is that I have a container with an  `AdMobBanner` at the bottom, and some other child components with flex layout. So I want to set the correct height on the `AdMobBanner` and resize the other components.

Actually I noticed that the `AdMobBanner` component has an `RNBanner` wrapped in a container `View` class, and the width and height are only being set on the `RNBanner`.

Maybe it would be better to set the width and height on the container `View`. That's basically what I'm trying to accomplish anyway. Something like this:

```
    return (
      <View style={{...this.state.style, ...this.props.style}}>
        <RNBanner
```

That way you could still override the width and height by passing your own props.

**Comment by alvaromb** [comment]
I have no context on this PR, can you give me some info @ndbroadbent? Is this PR still relevant? Thanks.

**Comment by ndbroadbent** [comment]
Hi @alvaromb, basically I just need to be able to dynamically resize my views when the ad is ready. I use flex layout, and without this change there is no way to dynamically resize my views.

**Comment by ndbroadbent** [comment]
Not sure what I did here, but apparently I confused GitHub:

```
ndbroadbent  wants to merge 2 commits into sbugert:master from unknown repository
```

Opening a new PR with these changes.

**Pull request closed without merging**